### PR TITLE
Finalize calib3d module split for fisheye part

### PR DESCRIPTION
**Pull request by asmorkalov**
### Pull Request Readiness Checklist

Continues: https://github.com/opencv/opencv/pull/18940

See details at https://github.com/opencv/opencv/wiki/How_to_contribute#making-a-good-pull-request

- [x] I agree to contribute to the project under Apache 2 License.
- [x] To the best of my knowledge, the proposed patch is not based on a code under GPL or another license that is incompatible with OpenCV
- [x] The PR is proposed to the proper branch
- [x] There is a reference to the original bug report and related work
- [x] There is accuracy test, performance test and test data in opencv_extra repository, if applicable
      Patch to opencv_extra has the same branch name.
- [x] The feature is well documented and sample code can be built with the project CMake

### CODE REVIEW

File: modules/3d/src/fisheye.cpp [hunk 1]
```diff
@@ -0,0 +1,592 @@
+#include "precomp.hpp"
```
Comment (vpisarev) [inline modules/3d/src/fisheye.cpp:1]: the license is missing

File: modules/3d/test/test_fisheye.cpp [hunk 2]
```diff
@@ -0,0 +1,384 @@
+#include "test_precomp.hpp"
```
Comment (vpisarev) [inline modules/3d/test/test_fisheye.cpp:1]: the license is missing

File: modules/stereo/test/test_geometry.cpp [hunk 3]
```diff
@@ -0,0 +1,343 @@
+#include "test_precomp.hpp"
```
Comment (vpisarev) [inline modules/stereo/test/test_geometry.cpp:1]: the license is missing

**Comment by vpisarev** [comment]
pls, add the missing license to the newly added files; otherwise, looks good for me :+1:

**Review by vpisarev — APPROVED** [review]

**Comment by asmorkalov** [comment]
Missing license: done.

**Pull request merged**